src/DataSource/ArrayDataSource.php

```diff
@@ -220,7 +220,10 @@ public function applyFilterDateRange($row, FilterDateRange $filter)
 
 		if ($values['from'] !== NULL && $values['from'] !== '') {
 			$date_from = \DateTime::createFromFormat($format, $values['from']);
-
+			if (!($date_from instanceof \DateTime)) {
```

Comment:
Try catch is for $row_value, this check is for eventuality that value in the data source is in different format than $format, which will cause exception on the return bellow